src/workspaces/features/UpdateDescriptionDialog/UpdateDescriptionDialog.tsx

```diff
@@ -0,0 +1,109 @@
+import Button from "core/components/Button";
+import Dialog from "core/components/Dialog";
+import Spinner from "core/components/Spinner";
+import useForm from "core/hooks/useForm";
+import { useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import Textarea from "core/components/forms/Textarea";
+import { useUpdateWorkspaceMutation } from "workspaces/graphql/mutations.generated";
+import { gql } from "@apollo/client";
+import { UpdateWorkspaceDescription_WorkspaceFragmentFragment } from "./UpdateDescriptionDialog.generated";
+import { UpdateWorkspaceError } from "graphql-types";
+
+type WorkspaceDescriptionDialogProps = {
+  onClose(): void;
+  open: boolean;
+  workspace: UpdateWorkspaceDescription_WorkspaceFragmentFragment;
+};
+
+type Form = {
+  description: string;
+};
+
+const WorkspaceDescriptionDialog = (props: WorkspaceDescriptionDialogProps) => {
```

Comment:
The name of the file and the name of the component is not the same. Could you align the name of the component based on the filename ,